cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -86,6 +86,7 @@ <h2 class="o-half-width-link-blob-group__header">
 {%- endblock %}
 
 {% block content_sidebar scoped -%}
+    {{ related_posts.render() }}
     {% for block in page.sidebar %}
```

Comment:
Hi Kurt Wall

I don't know why i am getting ccd on these emails..
. I no longer have the handle @kurt..... i think i see you have it now.
Kurt